src/compiler/crystal/codegen/target_machine.cr

```diff
@@ -19,7 +22,7 @@ module Crystal
           features += "+vfp2"
         end
       else
-        raise "Unsupported arch for target triple: #{target_triple}"
+        raise TargetMachine::Error.new("Unsupported architecture for target triple: #{target_triple} (only x86, x86_64, and ARM are supported)")
```

Comment:
I'm not found of hardcoding the link, I would go about adding a method with a maintained (or generated) list of supported targets, and use that instead..